src/main/java/com/agileavengers/icuconnectbackend/service/implementation/CommunityService.java

```diff
@@ -116,17 +108,15 @@ public CommunityDto getCommunity(Long id) {
         if (communityOptional.isPresent()) {
             return communityMapper.toDto(communityOptional.get());
         }
-        throw new ResponseStatusException(
-                HttpStatus.NOT_FOUND, "entity not found"
-        );
+        throw new ResponseStatusException(HttpStatus.NOT_FOUND, "entity not found");
     }
 
-//    @Override
-//    public Page<ReviewDto> getCommunityReviews(Long id, int page, int size) {
-//        Pageable pageable = PageRequest.of(page, size);
-//        Page<Review> reviewPage = reviewRepository.findAllByCommunity_Id(id, pageable);
-//        return reviewPage.map(reviewMapper::toDto);
-//    }
+    //    @Override
```

Comment:
Maybe this too? Or add a TODO so we don't forget?

Reply:
You are right. I removed it.